toJSONMap.js

```diff
@@ -0,0 +1,67 @@
+const JSONMap = {
```

Comment:
Similar to the comment I made in factory.js, you would also need to update the expected results here to be empty arrays, unfortunately. I do think it's worthwhile though.